jobs/wclouser-fxa-db-counts/wclouser_fxa_db_counts/main.py

```diff
@@ -0,0 +1,136 @@
+import click
+from datetime import datetime
+from google.cloud import bigquery
+
+
+@click.command()
+@click.option("--bq_project_id", help="GCP BigQuery project id", show_default=True, default="moz-fx-fxa-prod")
```

Comment:
Recognizing this is probably going to be replaced by something in bqetl: if this was planned to be running via airflow, `moz-fx-data-shared-prod` is a more appropriate project id to default to since airflow doesn't have `roles/bigquery.jobUser` access to the fxa projects.